test/src/e2e_vm_tests/test_programs/should_fail/contract_dependencies_conflicting_salt/contract_a/.gitignore

```diff
@@ -0,0 +1,2 @@
+out
+target
```

Comment:
I think we can remove the `.gitignore`s as the top-level `.gitignore` should cover these.